builtin/providers/aws/resource_aws_alb.go

```diff
@@ -292,7 +297,7 @@ func resourceAwsAlbUpdate(d *schema.ResourceData, meta interface{}) error {
 		} else if len(logs) == 0 {
 			attributes = append(attributes, &elbv2.LoadBalancerAttribute{
 				Key:   aws.String("access_logs.s3.enabled"),
-				Value: aws.String("false"),
+				Value: aws.String(strconv.FormatBool(false)),
```

Comment:
Good catch, I think I was initially going to convert everything to be a `aws.Bool` until I looked at the AWS SDK and noticed they wanted String types. Since a user will never be setting this, i'll go ahead and change this back!